history/titles/00_k_quelthalas.txt

```diff
@@ -37,11 +37,7 @@ c_shylfe_anore = {
 		liege=e_quelthalas
 	}
 	10.1.1={holder = 42002}
-	603.1.30={holder=42003}
-	604.1.1={
-		holder = 42075
-		liege = k_quelthalas
-	}
+	583.1.1={holder=patron310657} # Mathemarion Mognaur patreon	
```

Comment:
Shouldn't the patreon be in only one in one bookmark?
If it's like that then he'll be in all 3

Reply:
changed in c725d75db7866fb47146352ee930afd1c408fc65